Formula/pure.rb

```diff
@@ -0,0 +1,28 @@
+class Pure < Formula
+  desc "Pretty, minimal and fast ZSH prompt"
+  homepage "https://github.com/sindresorhus/pure"
+  url "https://github.com/sindresorhus/pure/archive/v1.15.0.tar.gz"
+  sha256 "58915e0317763bae87f2c3b5b8e643d761dc64004e1153c8f5ba72f658c7fd74"
+  license "MIT"
+
+  bottle :unneeded
+
+  depends_on "zsh-async"
```

Comment:
No such formula exists

Reply:
@SMillerDev As mentioned above… it `depends_on` #66167.

> **NOTE** that this `depends_on` #66167.